ResearchKit/Common/ORKScaleSliderView.m

```diff
@@ -83,15 +108,19 @@ - (instancetype)initWithFormatProvider:(id<ORKScaleAnswerFormatProvider>)formatP
         
         [self.slider addTarget:self action:@selector(sliderValueChanged:) forControlEvents:UIControlEventValueChanged];
     
+        leftRangeView.translatesAutoresizingMaskIntoConstraints = NO;
+        rightRangeView.translatesAutoresizingMaskIntoConstraints = NO;
         self.translatesAutoresizingMaskIntoConstraints = NO;
         self.slider.translatesAutoresizingMaskIntoConstraints = NO;
         self.leftRangeLabel.translatesAutoresizingMaskIntoConstraints = NO;
         self.rightRangeLabel.translatesAutoresizingMaskIntoConstraints = NO;
+        self.leftRangeImageView.translatesAutoresizingMaskIntoConstraints = NO;
+        self.rightRangeImageView.translatesAutoresizingMaskIntoConstraints = NO;
```

Comment:
Since `translatesAutoresizingMaskIntoConstraints` has been set on `leftRangeView`, no need to set it again.